docs/source/essentials/mutations.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Mutation
+---
+
+Now that we have discussed how to use a query to fetch data from a server using Apollo android, let's look into the other aspect of client-server communication - Sending and writing data to the server.
```

Comment:
Not everyone that reads this guide will be coming from last one, please change so that this guide can stand on its own

Reply:
Would something like ```Queries are useful to fetch data from a server, but client-server communication also requires sending and writing data to the server. This is where Mutations come handy.``` make sense? Or should this be removed at all?

Reply:
Sure!